src/containers/NahmiiWithdraw/index.js

```diff
@@ -0,0 +1,823 @@
+                    onFocus={() => this.onFocusNumberInput('gasPriceGweiInput')}
+                    onBlur={() => this.onBlurNumberInput('gasPriceGweiInput')}
+                  />
+                </FormItem>
+                <FormItem label={<FormItemLabel>{formatMessage({ id: 'gas_limit' })}</FormItemLabel>} colon={false}>
+                  <Input
+                    value={gasLimitInput}
+                    defaultValue={gasLimitInput}
+                    onChange={this.handleGasLimitChange}
+                    onFocus={() => this.onFocusNumberInput('gasLimitInput')}
+                    onBlur={() => this.onBlurNumberInput('gasLimitInput')}
+                  />
+                </FormItem>
+              </Panel>
+            </Collapse>
+            <DollarPrice>
+              {`1 ${assetToWithdraw.symbol} = ${formatFiat(assetToWithdrawUsdValue, 'USD')}`}
+            </DollarPrice>
+          </Form>
+        </div>
+        <div style={{ flex: 1, minWidth: '34rem' }}>
+          {
+            ongoingChallenges.get('details').length > 0 &&
+            <SettlementWarning
+              className="ongoing-challenges"
+              message={formatMessage({ id: 'challenge_period_progress' }, { staging_amount: totalStagingAmount.toString(), symbol: assetToWithdraw.symbol })}
+              description={formatMessage({ id: 'challenge_period_endtime' }, { endtime: moment(maxExpirationTime).format('LLLL'), symbol: assetToWithdraw.symbol })}
+              type="warning"
+              showIcon
+            />
+          }
+          {
+            settleableChallenges.get('details').length > 0 ?
+            (<SettlementWarning
+              className="confirm-settlement"
+              message={formatMessage({ id: 'settlement_period_ended' })}
+              description={
+                <div>
+                  <div>
+                    {formatMessage({ id: 'settlement_period_ended_notice' }, { symbol: assetToWithdraw.symbol, intended_stage_amount: totalSettleableStageAmount, tx_count: settleableChallenges.get('details').length })}
+                  </div>
+                  {
+                    TransferingStatus ?
+                      (
+                        <div style={{ marginTop: '2rem', display: 'flex', flexDirection: 'column' }} className="confirm-tx-status">
+                          {TransferingStatus}
+                        </div>
+                      ) : (
+                        <StyledButton className="confirm-btn" onClick={() => this.settle(assetToWithdraw)} disabled={disableConfirmSettleButton}>
+                          {formatMessage({ id: 'confirm_settlement' })}
+                        </StyledButton>
+                      )
+                  }
+                </div>
+              }
+              type="warning"
+              showIcon
+            />) : (
+              requiredSettlementAmount ?
+              (
+                <div className="start-settlement">
```

Comment:
I think it's possibly confusing for the user when the right side of the screen flicks like this, it's also not obvious to the user what a settlement is and why it's necessary. 

Do you think combining the withdraw and settlement summaries could work? 
I'm thinking something like this:
- put the 'Settle' button next to the 'Withdraw' button
- if the user has enough balance to withdraw without first settling, disable the Settle button and enable the Withdraw button. display a notice like "funds will be avaliable immediatly"
- if the user needs to settle first, disable Withdraw, enable Settle, display a notice explaining that they need to settle some funds

Reply:
Good idea. I agree this is better.